pkg/minikube/constants/constants.go

```diff
@@ -39,6 +40,9 @@ const MinikubeContext = "minishift"
 // MiniShiftEnvPrefix is the prefix for the environmental variables
 const MiniShiftEnvPrefix = "MINISHIFT"
 
+// Variable for registrator
+var Registrator registration.Registrator
```

Comment:
@praveenkumar this cannot work. The _start_ and _delete_ are in different executions. Different processes. And this is not a constant by any mean.